croquemort/webhook.py

```diff
@@ -0,0 +1,74 @@
+import logbook
+import requests
+
+from nameko.dependency_providers import Config
+from nameko.events import event_handler
+from nameko.utils.retry import retry
+
+from .logger import LoggingDependency
+from .storages import RedisStorage
+from .tools import is_url
+
+log = logbook.debug
+
+# See http://docs.python-requests.org/en/latest/user/advanced/#timeouts
+# We are waiting 3 sec for the connexion and 9 for the response.
+TIMEOUT = (3.1, 9.1)
+
+DELAY = 10
```

Comment:
Would there be an interest in qualifying what the values are about by adding a suffix?

With small numbers it can be slightly ambiguous to know what they are about.
Like `DELAY_INTERVAL`, `DELAY_TIMEOUT`, `DELAY_FACTOR`, `DELAY_VALUE` etc.